src/test/xmeta/testlexer.cpp

```diff
@@ -121,47 +104,51 @@ TEST_CASE("Spacing")
 
 TEST_CASE("Lexer uuid")
 {
-    std::string test_idl =
-        "namespace Windows.UI.ApplicationSettings \
-        { \
-            [contract(Windows.Foundation.UniversalApiContract, 1)] \
-            [uuid(b7de5527-4c8f-42dd-84da-5ec493abdb9a)] \
-            delegate void WebAccountProviderCommandInvokedHandler(WebAccountProviderCommand command); \
-        }";
+    std::istringstream test_idl{ R"(
+        namespace Windows.UI.ApplicationSettings
+        {
+            [contract(Windows.Foundation.UniversalApiContract, 1)]
+            [uuid(b7de5527-4c8f-42dd-84da-5ec493abdb9a)]
```

Comment:
Can we change a few characters on this UUID? I don't feel comfortable exposing a UUID that is in use now that I think about it.

Reply:
`uuidgen | clip`

Reply:
@kythant agreed. We can put it in a test-cleanup commit.